pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1632,6 +1631,11 @@ public CompletableFuture<Void> checkReplication() {
         }
 
         List<String> configuredClusters = topicPolicies.getReplicationClusters().get();
+        if (CollectionUtils.isEmpty(configuredClusters)) {
+            log.warn("[{}] No replication clusters configured", name);
```

Comment:
![image](https://github.com/apache/pulsar/assets/15684156/4fe622b9-28ae-4c03-a890-1b1ffc1e5378)
In our production env, because `configuredClusters` is empty, it caused the `Topic` to be deleted.

Reply:
Referring to the above flow diagram, method `onUpdate()->checkReplication()` is called before method `this.updateTopicPolicyByNamespacePolicy(policies)`.